theseus_gui/src/components/GridDisplay.vue

```diff
@@ -285,11 +295,11 @@ const filteredResults = computed(() => {
     </div>
     <section class="instances">
       <Instance
-        v-for="(instance, index) in instanceSection.value"
+        v-for="instance in instanceSection.value"
         ref="instanceComponents"
-        :key="instance.path"
+        :key="instance.path + instance.install_stage"
```

Comment:
is there a case where the instance path is not unique? why is this needed?

Reply:
The component needs to re-render when install stage is updated. IIRC it needs to do that because I forbid duplication on instances that arent done installing yet (for obvious reasons)

Reply:
hmm, this seems like a hacky workaround. i'll see if there's a better way when I do my commit